Dockerfile

```diff
@@ -41,12 +41,16 @@ RUN VLLM_TARGET_DEVICE="empty" python3 -m pip install /workspace/vllm/ --extra-i
 # In x86, triton will be installed by vllm. But in Ascend, triton doesn't work correctly. we need to uninstall it.
 RUN python3 -m pip uninstall -y triton
 
-# Install vllm-ascend
-RUN python3 -m pip install /workspace/vllm-ascend/ --extra-index https://download.pytorch.org/whl/cpu/
-
 # Install torch-npu
 RUN bash /workspace/vllm-ascend/pta_install.sh
 
+# Install vllm-ascend
```

Comment:
reminder: looks like we need to also update the install doc and CI to source set_env and LIBRARY_PATH

Reply:
This env is set within non-npu environment to make image build CI happy.  For doc and CI, we suppose users have NPU driver already.  

Is it a case to add more note into doc to let users know how to build/install vllm-ascend on a non-npu env?

Reply:
There was a note: https://vllm-ascend.readthedocs.io/en/latest/developer_guide/contributing.html#prepare-environment-and-build